app/views/notes/_title_suggestion.html.erb

```diff
@@ -0,0 +1,6 @@
+<blockquote>
+	<a href="/profile/<%= user %>"><%= user %></a> is suggesting an alternative title, "<%= title %>".
```

Comment:
Can we switch to 2-space indents here? sorry to be particular :-)

Also, can you share a screenshot of what this looks like?

Reply:
Oh I missed that!! I'll do!!

Reply:
Yes, I'll share the screenshots!